CHALLENGE.md

```diff
@@ -94,9 +92,43 @@ You are given a prepared SPA project that was bootstrapped with Create React App
 ![create-tournament-prompt](.github/create-tournament-prompt.png)
 
 - Clicking on the `CREATE TOURNAMENT` button will open a browser prompt with a message `Tournament Name:`, an input field to enter the tournament name and the buttons `Cancel` to cancel and `OK` to confirm.
-- When confirming, the tournament will be created on the fake REST API and added on the UI without any loading indicators.
+- The tournament name must contain only Latin letters, numbers, and spaces, not an empty string or only spaces.
+- When confirming, the tournament will be created immediately using an optimistic update in the UI and a fetch call on the fake REST API.
+
+### 6. Style it
+
+- The tournament has a border radius of `4px`.
+- The tournament name has the heading size `h6`.
+- The `Start` date must be displayed in the format `DD/MM/YYYY, HH:mm:ss` using the `en-GB` locale.
+- The horizontal spacing between the `EDIT` and `DELETE` button is `8px`.
+- The horizontal and vertical spacing between each tournament must be `24px`.
```

Comment:
Should we specify other spacings, for example between Tournament name and details, between details and buttons, etc?

It seemed a bit inconsistent for us to specify a few spacings but not others

Reply:
We try to find golden mean between being specific and overwhelming with requirements, if understand correctly.